packages/react/impression-area/src/useImpressionRef.tsx

```diff
@@ -2,73 +2,98 @@
 import { usePreservedCallback, useRefEffect } from '@toss/react';
 import { noop } from '@toss/utils';
 import debounce from 'lodash.debounce';
-import { useMemo, useRef } from 'react';
+import { useCallback, useEffect, useMemo, useRef } from 'react';
 import { ImpressionAreaProps } from './ImpressionArea';
 
 type Options = Pick<
   ImpressionAreaProps,
-  'rootMargin' | 'onImpressionStart' | 'onImpressionEnd' | 'timeThreshold' | 'areaThreshold' | 'root'
```

Comment:
Is this change intended?

Reply:
https://github.com/toss/slash/pull/283 
Previously, I created a pull request to add the "root" option.
I would appreciate it if you could check that pull request. 
(Cause of the problem: `iframe`)
@ragingwind